webrtc-stats.html

```diff
@@ -2352,11 +2353,11 @@ <h2>
                 </p>
                 <p>
                   Represents the total duration in seconds of all samples that have been produced
-                  by this source for the lifetime of this stats object. Can be used with
+                  by this source (excluding {{droppedSamplesDuration}}) for the lifetime of this stats object. Can be used with
```

Comment:
We should clarify that this is on the capture, the droppedSamples while have a consequence on the audioEnergy that is transmitted, it is not measured here.

Reply:
cc: @jan-ivar to confirm Firefox implementation.

Reply:
According to WPT, Firefox has not implemented media-source, so I think we just need to clarify that we should not measure energy of dropped samples